File: acs2/web/resources/vue-components/equipamento/portMappingInfo.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

var portMapping = function (p) {
    if (p) {
        this.externalPort = p.externalPort;
        this.internalClient = p.internalClient;
        this.internalPort = p.internalPort;
        this.portMapName = p.portMapName;
        this.enable = p.enable;
        this.protocol = p.protocol;
        this.remoteHost = p.remoteHost;
    }
}

File: acsAPI/src/main/java/model/device/portmapping/PortMappingInfo.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model.device.portmapping;

/**
 *
 * @author Le Agente
 */
public class PortMappingInfo {

    private String externalPort, internalClient, internalPort, portMapName, protocol, remoteHost;

    private Boolean enable;

    public PortMappingInfo() {
    }

    public String getExternalPort() {
        return externalPort;
    }

    public void setExternalPort(String externalPort) {
        this.externalPort = externalPort;
    }

    public String getInternalClient() {
        return internalClient;
    }

    public void setInternalClient(String internalClient) {
        this.internalClient = internalClient;
    }

    public String getInternalPort() {
        return internalPort;
    }

    public void setInternalPort(String internalPort) {
        this.internalPort = internalPort;
    }

    public String getPortMapName() {
        return portMapName;
    }

    public void setPortMapName(String portMapName) {
        this.portMapName = portMapName;
    }

    public String getProtocol() {
        return protocol;
    }

    public void setProtocol(String protocol) {
        this.protocol = protocol;
    }

    public String getRemoteHost() {
        return remoteHost;
    }

    public void setRemoteHost(String remoteHost) {
        this.remoteHost = remoteHost;
    }

    public Boolean getEnable() {
        return enable;
    }

    public void setEnable(Boolean enable) {
        this.enable = enable;
    }
}
